packer-plugin-docker.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: packer-plugin-docker
+  version: 1.0.10
+  epoch: 0
+  copyright:
+    - license: MPL-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/hashicorp/packer-plugin-docker
+      tag: v${{package.version}}
+      expected-commit: 0f58614d1b8fd31afa4ddf72e5440501d47bbdc0
+
+  - uses: go/bump
+    with:
+      deps: github.com/go-jose/go-jose/v3@v3.0.3 github.com/hashicorp/go-retryablehttp@v0.7.7 golang.org/x/crypto@v0.17.0 golang.org/x/net@v0.23.0 google.golang.org/grpc@v1.56.3 google.golang.org/protobuf@v1.33.0 github.com/hashicorp/packer-plugin-sdk@v0.5.4
+      replaces: github.com/go-jose/go-jose/v3=github.com/go-jose/go-jose/v3@v3.0.3 gopkg.in/square/go-jose.v2=github.com/go-jose/go-jose/v3@v3.0.3
+
+  - uses: go/build
+    with:
+      packages: .
+      output: packer-plugin-docker
+      ldflags: -X github.com/hashicorp/packer-plugin-docker/version.Version=${{package.version}}
+
```

Comment:
Does the binary respond to a --version or --help? Given it's a go binary it's highly likely, and we could at least add a basic test

Reply:
Thanks Mark, I've added the tests along with notes via https://github.com/kranurag7/os/commit/2aac0c27cc958983b36867cbad01a8053b5b9b92

Reply:
Thanks for following up. the absence of a --help or a --version, can you share info on how we were able to manually test this? It seems like it requires packer to operate, and I don't believe we have a packer apk today

Reply:
Have we been able to do any manual validation that the plugins are detected and usable by packer?